src/components/templates/TalkShowStudio/components/Audience/Audience.scss

```diff
@@ -0,0 +1,132 @@
+@import "scss/constants.scss";
+
+.Screenshare__audience {
+  &-title {
+    margin-top: $spacing--xl;
+    font-size: $font-size--lg;
+    line-height: 19px;
+    text-align: center;
+    color: rgba(255, 255, 255, 0.56);
+    &--hidden {
+      visibility: hidden;
+    }
+  }
+
+  &-container {
+    margin-top: $spacing--xl;
+    position: relative;
+    width: calc(35 * 30px);
```

Comment:
it's worth to use some global vars here

Reply:
Not sure if fixed elsewhere, but the &-container seems strange here too. Is it following BEM?